src/test/java/racingcar/domain/RacingCarsTest.java

```diff
@@ -4,22 +4,31 @@
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 
+import java.util.Arrays;
+import java.util.List;
+
 import static org.assertj.core.api.Assertions.assertThat;
 
 class RacingCarsTest {
 
     private RacingCars racingCars;
 
     @BeforeEach
-    void init() {
-        int countOfCar = 3;
-        racingCars = new RacingCars(countOfCar);
-    }
+    void setUp() {
+        List<String> carNames = Arrays.asList(
+                "pobi",
+                "crong",
+                "honux",
+                "YKJ"
+        );
+
+        racingCars = new RacingCars(carNames);
+    };
 
     @Test
     @DisplayName("자동차 경주 참가 테스트")
     void participate_racing_test() {
-        assertThat(racingCars.getParticipatingCars().size()).isEqualTo(3);
+        assertThat(racingCars.getParticipatingCars().size()).isEqualTo(4);
```

Comment:
assertj 검증문을 활용해도 좋을 것 같아요 ~!

```suggestion
        assertThat(racingCars.getParticipatingCars()).hasSize(4);
```

Reply:
수정해보겠습니다 😄